src/components/Icon/index.tsx

```diff
@@ -236,6 +236,9 @@ const ICONS = {
   credentials: () => (
     <path d="M4.364 9.333c-.804 0-1.455-.671-1.455-1.5 0-.828.651-1.5 1.455-1.5.803 0 1.454.672 1.454 1.5 0 .829-.651 1.5-1.454 1.5zm4.109-3c-.597-1.747-2.211-3-4.11-3C1.954 3.333 0 5.348 0 7.833c0 2.486 1.954 4.5 4.364 4.5 1.898 0 3.512-1.252 4.109-3h3.163v3h2.91v-3H16v-3H8.473z" />
   ),
+  database: () => (
```

Comment:
Small issue, icon does not seem to be centered

<img width="198" alt="Screenshot 2022-06-16 at 14 59 22" src="https://user-images.githubusercontent.com/2802867/174075137-0ba7d1d6-801a-438a-bbdd-8b0ed8c3cecd.png">